spec/rspec/rails/matchers/action_cable/have_broadcasted_to_spec.rb

```diff
@@ -12,13 +12,13 @@ def initialize(id)
       @id = id
     end
 
-    def to_global_id(options = {})
-      @global_id ||= GlobalID.create(self, :app => "rspec-suite")
+    def to_global_id(_options = {})
```

Comment:
Wondering why the cop complaining about not using `**options` doesn't come into play.